src/SFA.DAS.ApprenticeFeedback.Jobs/Functions/ApprenticeFeedbackSummaryTimerTrigger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using NServiceBus;
+using SFA.DAS.ApprenticeFeedback.Jobs.Domain.Messages.Commands;
+using SFA.DAS.ApprenticeFeedback.Jobs.Infrastructure;
+using System;
+
+namespace SFA.DAS.ApprenticeFeedback.Jobs.Functions
+{
+    public class ApprenticeFeedbackSummaryTimerTrigger
+    {
+        private readonly IFunctionEndpoint endpoint;
```

Comment:
"endpoint" instead of "_endpoint" naming convention isn't what we have in other projects (and this should also flag a naming convention warning in VS2022). Is there a reason for doing it this way?

Reply:
Have Updated across the various functions 👍